docs/blog/posts/Percona Monitoring and Management/pmm-3.0.0-beta.md

```diff
@@ -0,0 +1,29 @@
+---
+date: 2024-11-22
+description: >
+  Percona Monitoring and Management 3.0.0 has been released on Friday, November 22nd, 2024.
+authors: [catalinaadam]
+categories:
+  - Percona Monitoring and Management
+tags:
+  - Percona Monitoring and Management
+---
+
+# Percona Monitoring and Management 3.0.0 Beta has been released
+---
+<!-- more -->
+
+Percona Monitoring and Management 3.0.0 Beta has been released on Friday, November 22nd, 2024.
```

Comment:
Why do we have 22nd here while everywhere else we use the format November 22, 2024?

Reply:
fixed